src/modules/core/components/CalloutCard/CalloutCard.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { FormattedMessage, MessageDescriptor } from 'react-intl';
+
+import { ComplexMessageValues } from '~types/index';
+
+import { getMainClasses } from '~utils/css';
+import styles from './CalloutCard.css';
+
+const displayName = 'CalloutCard';
+
+interface Appearance {
+  theme?: 'primary' | 'info' | 'danger' | 'pinky';
+}
+
+interface Props {
+  /** Appearance object */
+  appearance?: Appearance;
+
+  /** label text */
+  label: MessageDescriptor | string;
+
+  /** Values for context text (react-intl interpolation) */
+  labelValues?: ComplexMessageValues;
+
+  /** A string or a `messageDescriptor` that make up the cards's content */
+  description?: MessageDescriptor | string;
+
+  /** Values for context text (react-intl interpolation) */
+  descriptionValues?: ComplexMessageValues;
+}
+
+const CalloutCard = ({
+  appearance = { theme: 'pinky' },
+  label,
+  labelValues,
+  description,
+  descriptionValues,
+}: Props) => {
+  return (
+    <div className={getMainClasses(appearance, styles)}>
+      <FormattedMessage {...label} values={labelValues} />
+      <FormattedMessage {...description} values={descriptionValues} />
```

Comment:
Both for `label` and for `description` your types give `string` as an option and you can't destructure a string. You either need to change types to restrict it to only `MessageDescriptor`, or you call it depending on what type the prop is (similar to some other core components)

Reply:
I think you should make them a `MessageDescriptor` required type. I'm not seeing this being used with simple strings anyway